nncf/torch/dynamic_graph/context.py

```diff
@@ -129,7 +130,8 @@ def maybe_add_node(self,
                        tensor_metas: List[Optional[TensorMeta]],
                        op_address: OperationAddress,
                        module_attrs: BaseLayerAttributes = None,
-                       ignored_algorithms: List[str] = None) -> Optional[DynamicGraphNode]:
+                       ignored_algorithms: List[str] = None,
+                       is_called_inside_nncf_module: bool = False) -> Optional[DynamicGraphNode]:
```

Comment:
Is it supposed to be merged on top of #1428? I see this change in both PR, it's a bit confusing.